_layouts/default.html

```diff
@@ -6,18 +6,11 @@
     <body>
       <div id="wrapper">
 
-		      <div id="header">
```

Comment:
oh yeah I took the footer out because I couldn't think of a good way to incorportate it into the design at the moment. this is completely a work in progress. thanks for the links!